lib/ae_mdw_web/views/aexn_view.ex

```diff
@@ -142,7 +147,7 @@ defmodule AeMdwWeb.AexnView do
       extensions: extensions,
       initial_supply: initial_supply,
       event_supply: event_supply,
-      holders: Stats.fetch_aex9_holders_count(state, contract_pk)
+      holders: num_holders
```

Comment:
perhaps we can use `nil` instead, since it would be a bit confusing for it to be either a string or a number

Reply:
The purpose is to catch user's attention as much as possible. But we can start with `nil` documenting it. If it continues to pop up like the balance before I change for `NA`.

Reply:
Actually another thing that would have being useful for a while, is to tag contracts as compliant or not. Opened here: https://github.com/aeternity/ae_mdw/issues/1410

Reply:
Agreed, that would be very useful